description/launch/zed_emulation.launch

```diff
@@ -9,7 +9,8 @@
             <remap from="disparity"             to="disparity/disparity_registered"/>
             <remap from="left/image_raw"        to="left/image_raw_color"/>
             <remap from="right/image_raw"       to="right/image_raw_color"/>
-            <param name="speckle_size"          value="400"/>
+            <param name="speckle_size"          value="0"/>
+            <param name="min_disparity"          value="0"/>
```

Comment:
does this help? I think speckly_size=0 sounds weird, and min_disparity=0 is typically bad data